src/styles/card.scss

```
@@ -0,0 +1,96 @@
+@import './variables';
+
+.card {
+  width: $card-width;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: $font-family;
+  background: $background-color;
+
+  &__image {
+    margin-top: 32px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    font-family: $font-family;
+    color: #060b35;
+    padding: 40px 16px 0;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    width: 100%;
+    text-align: left;
+    padding-left: 16px;
+    padding-top: 4px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin: 0;
+    padding: 8px 16px 0;
+    align-items: center;
+
+    &-label {
+      font-size: 12px;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      text-align: right;
+    }
+  }
+
+  &__stars {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    padding: 16px;
+    align-items: center;
+  }
+
+  &__reviews {
+    font-family: $font-family;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: right;
+    color: #060b35;
+  }
+
+  &__buy {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: $hover-color;
+    color: $background-color;
+    text-decoration: none;
+    width: $buy-width;
```

```suggestion
    width: 100%;
```
Consider setting paddings for the card block and the `widht: 100%;`

---

Do not work with widht: 100%